### PR TITLE
Updated cluster name 

### PR DESCRIPTION
**Pull request by mukrishn**
### Description
replaced unsupported characters from the cluster name

### CODE REVIEW

**Review by smalleni — APPROVED** [review]
LGTM

File: dags/openshift_nightlies/models/release.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def _get_git_user(self):
 
     def _generate_cluster_name(self):
         git_user = self._get_git_user()
-        git_branch = environ['GIT_BRANCH']
+        git_branch = str(environ['GIT_BRANCH']).replace("_","-")
```
Comment (afcollins) [inline dags/openshift_nightlies/models/release.py:37]: Cool. Do we also need to lower-case any upper-case?

File: dags/tests/openshift_nightlies/util/test_manifest.py [hunk 2]
```diff
@@ -59,7 +59,7 @@ def test_manifest_arm(self,mocked_manifest):
 
     def test_cloudreleases_amd(self,mocked_releases):
         releases = mocked_releases
-        assert len( releases ) == 145
+        assert len( releases ) == 143
```
Comment (afcollins) [inline dags/tests/openshift_nightlies/util/test_manifest.py:62]: honestly this is a flakey assertion. You can probably remove both instead of changing.

**Review by jtaleric — APPROVED** [review]
lgtm

File: dags/openshift_nightlies/models/release.py (hunk 1 above)
Comment (jtaleric) [reply dags/openshift_nightlies/models/release.py:37]: 🤔

Comment (dcbw) [reply dags/openshift_nightlies/models/release.py:37]: Yes, I think so.

**Comment by jtaleric** [comment]
:shipit:

**Pull request merged**